### PR TITLE
[FIX] fix confirms of carts

### PR DESCRIPTION
**Pull request by YisusJe**

### CODE REVIEW

File: src/components/Cart/index.js [hunk 1]
```diff
@@ -472,16 +466,6 @@ const CartUI = (props) => {
             />
           )}
         </CartSticky>
-        <Confirm
```
Comment (carlosq-ordering) [inline src/components/Cart/index.js:475]: @YisusJe  why are you deleting this component?

**Review by carlosq-ordering — CHANGES_REQUESTED** [review]

File: src/components/Cart/index.js (hunk 1 above)
Comment (YisusJe) [reply src/components/Cart/index.js:475]: Because was duplicate by accident

Comment (carlosq-ordering) [reply src/components/Cart/index.js:475]: @YisusJe It's ok this title ?
image

Comment (YisusJe) [reply src/components/Cart/index.js:475]: let me change it for offers or products

**Review by carlosq-ordering — APPROVED** [review]

**Pull request merged**